src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/MSBuildAPIUtility.cs

```diff
@@ -438,7 +440,9 @@ internal static bool IsPackageReferenceProject(Project project)
                             };
                         }
 
-                        installedPackage.ResolvedVersion = library.Version;
+                        installedPackage.ResolvedPackageMetadata = PackageSearchMetadataBuilder.FromIdentity(
+                            new PackageIdentity(library.Name, library.Version)).Build();
```

Comment:
very minor, but I would prefer formatting like this:
```csharp
                        installedPackage.ResolvedPackageMetadata = PackageSearchMetadataBuilder
                            .FromIdentity(new PackageIdentity(library.Name, library.Version))
                            .Build();
```

I think breaking a method call over multiple lines makes it harder to read, so prefer to keep it on oneline, unless the parameter list is too long. Also when using "fluent" APIs, having each call on a new line makes it easier to see the flow.

If you change this, also see a few lines below for another instance.

Reply:
Fixed in https://github.com/NuGet/NuGet.Client/pull/2928/commits/f14ddc33891ac566e32eb2f1ce48b766a3385216